src/integration/resources/docker/config/prometheus.yml

```diff
@@ -0,0 +1,48 @@
+# my global config
+global:
+  external_labels:
+    role: "remote"
+  scrape_interval:     5s # Set the scrape interval to every 5 seconds. Default is every 1 minute.
+  evaluation_interval: 5s # Evaluate rules every 5 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+
+# Alertmanager configuration
+alerting:
+  alertmanagers:
+  - static_configs:
+    - targets:
+      # - alertmanager:9093
```

Comment:
Nit: if we don't want these, can we remove and put `null` instead?

Reply:
Oh uh, this file actually should be a `git mv` (i.e. I didn't really touch it), apologies. Let me clear that up.